client/Makefile

```diff
@@ -7,21 +7,34 @@
 
 NAME	=	myteams_cli
 
+SRC_COMMANDS	=	src/get_commands/get_cmd_unsubscribe_teams.c	\
+					src/get_commands/get_cmd_subscribe_teams.c	\
+					src/get_commands/get_cmd_logout.c	\
+					src/get_commands/get_cmd_login.c	\
+					src/get_commands/get_cmd_send_message.c	\
+					src/get_commands/get_cmd_use.c	\
+					src/get_commands/get_cmd_create.c	\
+					src/get_commands/get_cmd_info.c	\
+					src/get_commands/get_cmd_list.c	\
+					src/get_commands/get_cmd_messages.c	\
+					src/get_commands/get_cmd_user.c	\
+					src/get_commands/get_cmd_users.c
+
+
+
 SRC	=	main.c\
 		src/client.c\
 		src/manage_commands/manage_command.c	\
-		src/get_commands/get_cmd_login.c	\
-		src/get_commands/get_cmd_logout.c	\
 		src/get_commands/check_params.c	\
 		src/get_commands/manage_params.c	\
-		src/get_commands/get_cmd_send_message.c	\
 		src/manage_commands/make_command_rfc_compatible.c	\
 		src/manage_commands/parse_server_response.c \
-		src/manage_commands/reply_codes.c
+		src/manage_commands/reply_codes.c	\
+		src/manage_commands/send_command.c
```

Comment:
What is the difference between them and the ones above? They both handle commands